server/tests/index.js

```diff
@@ -0,0 +1,3 @@
+import users from './users';
```

Comment:
I get the feeling that you did this in order to make the tests run in the order in which they were arranged.

Again, writing unit tests would not require you doing this since each test runs independent of the other tests in the whole test file.
So, look into redrafting your tests to be more unit-tests like.